### PR TITLE
Add Documentation-Updates.md

### PR DESCRIPTION
**Pull request by rhodh**
Detail where and how we keep our documentation up to date in particular SO.

### CODE REVIEW

File: docs/Ways-of-Working/Communication/Documentation-Updates.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Documentation Updates
+
+Documentation at AM is an ongoing project, we're currently focused on improving documentation and making it for everyone to access and discover information to help their understanding of how we work, and the platforms we work on.
```
Comment (91dave) [inline docs/Ways-of-Working/Communication/Documentation-Updates.md:3]: An ongoing ~project~ *process*, maybe?

Also - presumably missing the word "easier"?
> ...and making it **easier** for everyone...

Comment (rhodh) [reply docs/Ways-of-Working/Communication/Documentation-Updates.md]: Thanks Fixed!

**Review by 91dave — APPROVED** [review]
Approving after wording comments were addressed.

File: docs/Ways-of-Working/Communication/Documentation-Updates.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Documentation Updates
+
+Documentation at AM is an ongoing process, we're currently focused on improving documentation and making it easier for everyone to access and discover information to help their understanding of how we work, and the platforms we work on.
+
+!!! Tip 
+    Documentation is a living document and is the responsibility of the whole department to improve and update the documentation, don't just leave it for the next person to do.
+
+
+We have 3 locations where we store documentation as a department:
```
Comment (edwardearle) [inline docs/Ways-of-Working/Communication/Documentation-Updates.md:9]: I think this is starting to look like the documentation directory in the KB. Which also highlights that there are more than 3 places, and loads more types of documentation in many other places.

I would suggest that the Playbook expresses the fact that documentation is a vital part of communication and collaboration, and points to the KB documentation directory to understand current tools for each type.

The KB doc directory could then be updated to contain all these things, pointing out their location, purpose, and and contribution guidelines (or signposting to more in depth guidelines)

**Review by edwardearle — CHANGES_REQUESTED** [review]

**Pull request closed without merging**